migrations/gateway_migration.go

```diff
@@ -0,0 +1,16 @@
+// +build gateway
+
+package migrations
+
+import (
+	"github.com/loomnetwork/transfer-gateway/builtin/plugins/gateway"
+)
+
+func GatewayMigration(ctx *MigrationContext) error {
```

Comment:
You also need to hook this up in https://github.com/loomnetwork/loomchain/blob/734f1cfa184cd43390df06883f8f5a9cbc62220c/cmd/loom/loom.go#L834